### PR TITLE
feat(Charts): adding color and units customization

### PR DESCRIPTION
**Pull request by nbaenam**
After talking with Sebastián Galiano, I've added _Color customization_ and _Units customization_ to the [Customize your charts](https://docs.newrelic.com/docs/query-your-data/explore-query-data/use-charts/use-your-charts#customize-charts) section.

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 1]
```diff
@@ -164,18 +164,18 @@ Dashboards features include:
 >
     You can customize the color of your charts per series in area, bar, line, pie, and stacked bar charts.
   
-   To customize the color per series:
+    To customize the color per series:
     1. Run your query.
     1. Open the Colors menu in the chart settings.
     1. Select the plus <Icon name="fe-plus"/> icon.
     1. Select the series you want to change the color.
     1. Select the desired color.
```
Comment (x8a) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:172]: Should we change this to consecutive numbers?

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx [hunk 2]
```diff
@@ -126,8 +130,57 @@ While we try our best to optimize how we display your data, sometimes you may ha
...
+    title="Color customization"
+>
+    You can customize the color of your charts per series in area, bar, line, pie, and stacked bar charts.
+  
+    To customize the color per series:
+    1. Run your query.
+    1. Open the Colors menu in the chart settings.
+    1. Select the plus <Icon name="fe-plus"/> icon.
+    1. Select the series you want to change the color.
+    1. Select the desired color.
```
Comment (x8a) [inline src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:145]: Same as above, should we change this to consecutive numbers?

**Review by x8a — APPROVED** [review]

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx]: Yes, I'm going to change it. Thanks!

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx]: Yes, I'm going to change it. Thanks!

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx [hunk 3]
```diff
@@ -126,8 +130,57 @@ While we try our best to optimize how we display your data, sometimes you may ha
...
+    title="Units customization"
+>
+    You can customize the unit on your Y axis and in each of your series. It is available in bar, line, and stacked charts. 
+        
+    To customize the unit on your Y axis:
+    1. Run a time series query.
+    1. Open the units menu in the customizations UI.
+    1. Use the plus <Icon name="fe-plus"/> icon to add a per series customization.
+    1. Select the series that needs another unit.
+    1. Select the unit.
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx]: Yes, I'm going to change it. Thanks!

**Review by x8a — APPROVED** [review]
👌🏻

**Pull request merged**